src/features/operator/operator.api.ts

```diff
@@ -809,22 +816,27 @@ export const operatorApi = tdexApi.injectEndpoints({
       providesTags: ['Trade'],
     }),
     // TODO: fix wrong calculation
-    totalCollectedSwapFees: build.query<number, Market.AsObject[] | undefined>({
-      queryFn: async (arg, { getState }) => {
+    totalCollectedSwapFees: build.query<
+      number,
+      { markets: Market.AsObject[] | undefined; prices: any; assets: any; network: any }
+    >({
+      queryFn: async ({ markets, prices, assets, network }, { getState }) => {
         const state = getState() as RootState;
         const client = selectOperatorClient(state);
         const metadata = selectMacaroonCreds(state);
         return retryRtkRequest(async () => {
-          if (!arg) return { data: 0 };
+          if (!markets) return { data: 0 };
           let totalCollectedSwapFees = 0;
-          const markets = arg.map(({ baseAsset, quoteAsset }) => {
+          console.log({ markets });
+          const marketsJSPB = markets.map(({ baseAsset, quoteAsset }) => {
```

Comment:
What `JSPB` stands for?

Reply:
markets is just a simple array containing [{baseAsset, quoteAsset}]

marketsJSPB is an array containing the google js protobuff markets object